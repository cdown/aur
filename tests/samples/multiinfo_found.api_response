{"version":1,"type":"multiinfo","resultcount":2,"results":[{"ID":229275,"Name":"tzupdate","PackageBaseID":99473,"PackageBase":"tzupdate","Version":"0.4.0-2","Description":"Set the system timezone based on IP geolocation","URL":"http:\/\/github.com\/cdown\/tzupdate","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1441577921,"LastModified":1441747990,"License":"ISC","URLPath":"\/cgit\/aur.git\/snapshot\/tzupdate.tar.gz","CategoryID":1,"Popularity":0},{"ID":229289,"Name":"yturl","PackageBaseID":68930,"PackageBase":"yturl","Version":"1.20.0-2","Description":"YouTube videos on the command line","URL":"http:\/\/github.com\/cdown\/yturl","NumVotes":1,"OutOfDate":0,"Maintainer":"cdown","FirstSubmitted":1365426996,"LastModified":1441751184,"License":"MIT","URLPath":"\/cgit\/aur.git\/snapshot\/yturl.tar.gz","CategoryID":1,"Popularity":0}]}