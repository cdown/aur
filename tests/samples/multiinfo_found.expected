[
    {
        "category_id": 1,
        "description": "Set the system timezone based on IP geolocation",
        "first_submitted": 1441577921,
        "id": 229275,
        "last_modified": 1441747990,
        "license": "ISC",
        "maintainer": "cdown",
        "name": "tzupdate",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "tzupdate",
        "package_base_id": 99473,
        "popularity": 0,
        "url": "http://github.com/cdown/tzupdate",
        "url_path": "/cgit/aur.git/snapshot/tzupdate.tar.gz",
        "version": "0.4.0-2"
    },
    {
        "category_id": 1,
        "description": "YouTube videos on the command line",
        "first_submitted": 1365426996,
        "id": 229289,
        "last_modified": 1441751184,
        "license": "MIT",
        "maintainer": "cdown",
        "name": "yturl",
        "num_votes": 1,
        "out_of_date": false,
        "package_base": "yturl",
        "package_base_id": 68930,
        "popularity": 0,
        "url": "http://github.com/cdown/yturl",
        "url_path": "/cgit/aur.git/snapshot/yturl.tar.gz",
        "version": "1.20.0-2"
    }
]